tests/unit/config/test_config_loaders.py

```
@@ -188,6 +188,79 @@ def test_load_lintro_global_config_returns_empty_when_no_globals() -> None:
+
+    with mock_lintro_section(mock_data):
+        result = load_lintro_global_config()
+
+    assert_that(result).is_equal_to({"global_setting": "value"})
+    assert_that(result).does_not_contain_key("post_checks")
+    assert_that(result).does_not_contain_key("module_size")
+    assert_that(result).does_not_contain_key("versions")
+
+
+def test_unknown_key_remains_global(
+    mock_lintro_section: Any,
```

Ruff (0.15.20)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `mock_lintro_section`

(ANN401)